index.html

```diff
@@ -18,19 +18,31 @@
             <header>
                 <nav>
                     <a href="#" class="my-logo">My Logo</a>
-
+                    
                     <button class="menu">
                         <p class="Menu-Icon"></p>
                         <p class="Menu-Icon"></p>
                         <p class="Menu-Icon"></p>
                     </button>
+
+                    <button class="Portoflio nav-button">
+                        Portoflio
+                    </button>
+
+                    <button class="About-b nav-button">
+                        About
+                    </button>
+
+                    <button class="Contact nav-button">
+                        Contact
+                    </button>
```

Comment:
- [ ] [OPTIONAL] I would strongly suggest using the `<a>` tag instead of the `<button>` for your navigation links and also adding the `<href>` values so the users can navigate in your portfolio and make it a better experience (UX) :wink: :raised_hands: